frontend/src/components/custom-tools/document-manager/DocumentManager.jsx

```diff
@@ -92,22 +92,24 @@ function DocumentManager({ generateIndex, handleUpdateTool, handleDocChange }) {
 
   useEffect(() => {
     if (isSimplePromptStudio) {
-      items[0] = {
-        key: "1",
-        label: (
-          <Tooltip title="PDF View">
-            <FilePdfOutlined />
-          </Tooltip>
-        ),
-      };
-      items[1] = {
-        key: "2",
-        label: (
-          <Tooltip title="Raw View">
-            <FileTextOutlined />
-          </Tooltip>
-        ),
-      };
+      items = [
```

Comment:
@tahierhussain Small doubt here, why are we not adding this to the top defined place where `items` is initialized, seems like we are not adding anything sensitive here and since its just a label; modification ?

Reply:
@athul-rs We need to render the modified items specific to Simple Prompt Studio. We cannot add them at the top where we're initializing `items`. Since this component will be used in both Main and Simple Prompt Studio, we can only determine inside the function whether the current prompt studio is Simple or Main (using `isSimplePromptStudio` state).

Reply:
We could improve the part of code and avoid `useEffect` here, but upon discussion this is not staright-forward change and by design we'll keep go forward with this for now.